server/config/persist_options.go

```diff
@@ -200,6 +200,13 @@ const (
 	hotRegionScheduleLimitKey      = "schedule.hot-region-schedule-limit"
 	schedulerMaxWaitingOperatorKey = "schedule.scheduler-max-waiting-operator"
 	enableLocationReplacement      = "schedule.enable-location-replacement"
+	// it's related to schedule, but it's not an explicit config
+	// todo: do we need this prefix??
+	disableTikvSplitRegion = "schedule.disable-tikv-split-region"
```

Comment:
better to `enable-xxxx`, and default value is true.

Reply:
https://github.com/tikv/pd/pull/5485/commits/ad4539f224b59c327481ff25f89e85f7046cd8e6